src/main/java/com/github/maxopoly/MemeMana/MemeManaDAO.java

```diff
@@ -29,9 +36,9 @@ private void prepareMigrations() {
 		registerMigration(
 				0,
 				false,
-				"CREATE TABLE IF NOT EXISTS manaUnits (id int not null, baseAmount double not null, fillGrade float not null default 1.0,"
-						+ "date datetime NOT NULL default NOW(), ownerId int NOT NULL, unique (id), index `ownerIdIndex` (owner_id));",
-				"create table if not exists manaOwners (id int not null, name varchar(36) not null, type int not null, primary key(id));");
+				"CREATE TABLE IF NOT EXISTS manaUnits (id int not null, baseAmount double not null, fillGrade double not null default 1.0,"
+						+ "date datetime NOT NULL default NOW(), ownerId BIGINT NOT NULL, unique (id), index `ownerIdIndex` (ownerId));",
+				"create table if not exists manaStats (id BIGINT not null, streak int not null, lastDay BIGINT not null, primary key(id));");
```

Comment:
The owner table is needed, because I intend to allow groups to own mana (not only players). The table you added is needed for tracking inidividual players streaks, but we need an owner table as well. 

Tracking in the manaStats table should use a varchar(36) (which is the length of a written out uuid) to identify players, not a long. Otherwise you need to join every time you want to update a streak, which isnt necessary.

I'll probably have separate tables for groups/players each to map them to an int id, which will be used internally as the ownerId used in manaUnits.

After that the ownerId in the manaUnit table can also stay an int, we're not gonna have more than 2 billion players/groups.

Reply:
All the stuff with mana owners will eventually be replaced with AltManager's thing, whatever that ends up being. It can also abstract over groups vs players, so it'll all work out in the end.

The reason things are longs right now is they are using BanStick pid's to stay consistent and specifically to avoid having to write out UUID's as strings.

Reply:
Ok sounds good